Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -118,26 +144,26 @@ void FieldProbe::ComputeDiags (int step)
         const auto prob_hi = gm.ProbHi();
 
 #if (AMREX_SPACEDIM == 2)
-        m_probe_in_domain = x_probe >= prob_lo[0] and x_probe < prob_hi[0] and
-                            z_probe >= prob_lo[1] and z_probe < prob_hi[1];
+        m_probe_in_domain = probe1.pos(0) >= prob_lo[0] and probe1.pos(0) < prob_hi[0] and
+                            probe1.pos(2) >= prob_lo[1] and probe1.pos(2) < prob_hi[1];
 #else
-        m_probe_in_domain = x_probe >= prob_lo[0] and x_probe < prob_hi[0] and
-                            y_probe >= prob_lo[1] and y_probe < prob_hi[1] and
-                            z_probe >= prob_lo[2] and z_probe < prob_hi[2];
+        m_probe_in_domain = probe1.pos(0) >= prob_lo[0] and probe1.pos(0) < prob_hi[0] and
+                            probe1.pos(1) >= prob_lo[1] and probe1.pos(1) < prob_hi[1] and
+                            probe1.pos(2) >= prob_lo[2] and probe1.pos(2) < prob_hi[2];
 #endif
         if(lev == 0) m_probe_in_domain_lev_0 = m_probe_in_domain;
 
         if( !m_probe_in_domain ) break;
```

Comment:
Note - because of this `break`, the code setting `m_data` for higher levels won't be reached if the probe point is not in the refined patch. This can be fixed by using the previous method, wrapping the code calculating `fp_values` in an `if` block.

Reply:
Thanks for the hint!

We replace this line with:
```suggestion
        if( m_probe_in_domain ) {
```
and close the `}` after the MPI comm part again (`amrex::ParallelDescriptor::Recv`).